hack/make/deploy/deploy.mk

```diff
@@ -33,6 +33,13 @@ undeploy:
 	helm uninstall dynatrace-operator \
 			--namespace dynatrace
 
+## Remove all Dynatrace Operator resources from the cluster and node filesystem
+cleanup: cleanup/cluster cleanup/node-fs
+
 ## Remove all Dynatrace Operator resources from the cluster
-cleanup:
+cleanup/cluster: 
```

Comment:
So to me LGTM but one thing: should we move it to a single .mk file? Don't have the strongest opinion here just an idea.